### DISCUSSION TITLE
bundle app for backup - problem with windows store app?

### DISCUSSION THREAD

**Discussion by kwanice** [Devolutions/UniGetUI#3126] [Q&A] [answered]
Hello all, 
i have a question regarding the creation of bundle app, for backup and future fresh install.
i saw there is a difference in a source between the winget:msstore and Microsoft Store 
i wanted to create a bundle of all my app, but the app i have installed directly from the windows store have the source "microsoft store" and tose which i install via winget have the source "winget:msstore"
the problem is when i want to open the bundle i have created, only the file with the source "winget:msstore" can be restored not the one with "microsoft store"
=> how can i do to have a proper backup ?

ps : i have seen that if i overwrite the installation of the microsoft store app with winget, the bundle i will create will be good.
but i have lot of app direclty installed via Microsoft store...

any idea?

thanks !

**Comment by kwanice** [comment]
![2024-12-24 Screenshot 133022](https://github.com/user-attachments/assets/f8f0e2ae-96b9-4b11-9898-718aea658da9)
![2024-12-24 Screenshot 133030](https://github.com/user-attachments/assets/c204e1ee-9944-4833-8aaa-54203a9d0eca)

just to show u the difference

the second one with winget will be fully backup and easy to use for a fresh install

**Comment by marticliment** [comment, marked as answer]
Hello @kwanice,

Yes, you have the proper backup. While the app you have installed can be installed via WinGet: msstore, Microsoft Store apps have certain limitations that prevent WinGet from properly detecting them, and hence they show as Microsoft Store, and can't get imported when using UniGetUI's bundles

**Reply by kwanice** [reply]
thanks, i reinstalled with the 'winget' installation (just need to find the package and click to install it, 5 sec and its done)
but i feel, im not dev, it should not be so difficult to switch the Microsoft store installation to the winget version
i did manually, its quite boring lol

**Reply by marticliment** [reply]
This is something for what I am working on a fix, but since this is a winget thing and not an unigetui one, it is hard to fix it from here

**Reply by kwanice** [reply]
thanks for replying and tahnks for your hard work, ur app is great by the way
i did it manually so its fine also :) now i will use only unigetui to install so no problem :)
